### DISCUSSION TITLE
Running Jupyter Notebooks in PubPub

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#1845] [Show and tell]
Hi all,

just thought I'd quickly share this here - for those interested in using Jupyter Notebooks for their Pubs, I can confirm that these can be embedded using **JupyterLite** (shoutout to Tony Hirst, how just recently [blogged about this](https://blog.ouseful.info/2022/03/03/embedding-jupyterlite-in-browser-notebooks-in-documentation-and-online-educational-materials/) browser-based implementation of Jupyter Notebooks).

Test-Pub here: [https://copim.pubpub.org/pub/jupyterlite-in-pubpub/](https://copim.pubpub.org/pub/jupyterlite-in-pubpub/)

**Comment by metasj** [comment]
This is great.  I wonder how this could work with [Curvenote](https://curvenote.com/docs/overview/projects) and pubs hosted on [2i2c.](https://2i2c.org/)..
